encode.go

```diff
@@ -444,6 +444,10 @@ func formatTs(t time.Time) (b []byte) {
 			return []byte("infinity")
 		}
 	}
+	return FormatTimestamp(t)
+}
+
+func FormatTimestamp(t time.Time) []byte {
```

Comment:
If this is to be a public function, it should probably be documented (as should `ParseTimestamp`--I missed that originally) and tested. The existing tests cover this okay I think, but do you mind adding docs?